rabbitmq.yml

```diff
@@ -22,11 +28,13 @@ services:
       # docker publish port 5555 to 5555 
       # (將 docker 內部 ip 5555, 跟外部 5555 做連結)
       - 5555:5555
-    depends_on:
```

Comment:
https://github.com/FinMind/FinMindBook/blob/master/DataEngineering/Chapter7/7.6/rabbitmq.yml#L27

it seems that `flower` still depends on `rabbitmq`